### PR TITLE
docs(community): adds new GrubHub logo to Community page

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

**Comment by dorbin** [comment]
...and realphabetize

File: community/index.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ makerbot_user:
 cleartax_in_user:
   image_path: assets/images/users/cleartax_in_logo.png
   alt: "ClearTax.in Logo"
+grubhub_user:
+  image_path: assets/images/users/grubhub_logo.png
+  alt: "GrubHub Logo"
```
Comment (demobox) [inline community/index.md]: `Grubhub`? See e.g. the footer of https://www.grubhub.com

**Review by ghost — APPROVED** [review]

**Review by demobox — COMMENTED** [review]
Looks good to me - thanks for updating the capitalization, Dave!

**Comment by dorbin** [comment]
#616

**Pull request merged**